adapters/repos/db/repo.go

```diff
@@ -170,5 +196,26 @@ func (d *DB) Shutdown(ctx context.Context) error {
 		}
 	}
 
+	d.shutDownWg.Wait() // wait until job queue shutdown is completed
```

Comment:
Does this waitgroup have to be part of the `*DB` struct? Could this also just be created at the beginning of this method? (Assuming this is the only place it's used.)

Reply:
it is used in the worker function to signal that the shutdown is complete